modules/amazon/index.js

```diff
@@ -0,0 +1,60 @@
+var amazon = require('amazon-product-api');
+var amazonClient = null;
+
+module.exports.init = function(bot) {
+  bot.getConfig("amazon.json", function(err, conf) {
+    if(err) bot.say("Unable to load amazon module: " + err);
```

Comment:
I think these should move to `bot.log.error`.

For people who don't have amazon.json, it's fine to log an error but not spam their channel.